packages/components/ovh-shell/package.json

```diff
@@ -11,17 +11,20 @@
   "author": "OVH SAS",
   "sideEffects": "false",
   "type": "module",
-  "exports": {
-    ".": "./dist/index.js",
-    "./client": "./dist/client/index.js",
-    "./plugin/i18n": "./dist/plugin/i18n/index.js"
-  },
   "main": "./dist/index.js",
   "scripts": {
-    "build": "tsc"
+    "build": "tsc",
+    "dev": "tsc -w"
```

Comment:
i try to found `-w` option into the typescript compiler documentation but nothing.
is't sure this option is available ?

I found this `tsc --watch`

Reply:
It's a shorthand

```sh
$ npx tsc --help
Version 3.8.3
Syntax:   tsc [options] [file...]

Examples: tsc hello.ts
          tsc --outFile file.js file.ts
          tsc @args.txt
          tsc --build tsconfig.json

Options:
 -h, --help                                         Print this message.
 -w, --watch                                        Watch input files.
```

;)